connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -171,9 +173,9 @@ public void startConnector(ConnectorConfig connConfig, ConnectorContext ctx, Con
         WorkerConnector workerConnector = new WorkerConnector(connName, connector, ctx, statusListener);
 
         log.info("Instantiated connector {} with version {} of type {}", connName, connector.version(), connClass.getName());
-        workerConnector.initialize();
+        workerConnector.initialize(connConfig.originalsStrings());
```

Comment:
This invokes connector code and `WorkerConnector.initialize()` doesn't look like it catches & processes those errors. This is unlikely to be an issue in practice since it has a default implementation and most connectors won't override it, but this should still probably be in a `try`/`catch`.